### PR TITLE
Adding support for LED Bulb by Lonhonso ZB-RGBCW

### PR DESCRIPTION
**Pull request by TSergeymsk**
Adding support for LED Bulb by Lonhonso

### CODE REVIEW

File: devices.js [hunk 1]
```diff
@@ -2302,6 +2302,13 @@ const devices = [
             await configureReporting.onOff(endpoint);
         },
     },
+    {
+        zigbeeModel: ['ZB-RGBCW'],
+        model: 'Zigbee 3, RGW+W+C',
```
Comment (Koenkk) [inline devices.js:2307]: Please fill in a meaningful model here, preferably something that can be Googled and leads to this bulb.

Comment (TSergeymsk) [reply devices.js:2307]: Unfortunately, there is no meaningul model in the paper manual nor in the Aliexpress lot. Just "ZigBee 3.0 Tuya Smart Led Lamp Bulb Light E27 220V RGB+W+C" in description.
I've seen several mentions of id ZB-RGBCW on git, but without any model number.

Comment (Koenkk) [reply devices.js:2307]: No problem, can you provide a picture of the device?

Comment (TSergeymsk) [reply devices.js]: Sure.
ZB-RGBCW

Comment (Koenkk) [reply devices.js]: Thanks!

**Pull request merged**